### PR TITLE
Empty other warden scopes upon login

### LINKED ISSUE #4656: Empty other warden scopes upon login
To finally fully disable "multilogin" and related errors, as we don't support multiple accounts.

### PR DESCRIPTION
**Pull request by Kartones**
Closes #4656

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5344/))

**Comment by Kartones** [comment]
@juanignaciosl  CR please

Tested locally and at staging and working fine.

As talked with @javisantana we only want this behaviour upon user login, and only at the editor, so no need to touch at other places.

File: config/initializers/warden.rb [hunk 1]
```diff
@@ -132,5 +132,19 @@ def authenticate!
 # @see ApplicationController.update_session_security_token
 Warden::Manager.after_set_user except: :fetch do |user, auth, opts|
   auth.session(opts[:scope])[:sec_token] = Digest::SHA1.hexdigest(user.crypted_password)
+
+  # Only at the editor, and only after new authentications, destroy other sessions
+  # @see #4656
+  warden_proxy = auth.env['warden']
+  # On testing there is no warden global so we cannot run this logic
+  if warden_proxy
+    auth.env['rack.session'].select { |key, value|
```
Comment (juanignaciosl) [inline config/initializers/warden.rb]: I don't know if this is guaranteed to be present. If not, a `.fetch('rack.session', [])` saves a `nil` check.

**Comment by juanignaciosl** [comment]
LGTM, :es: if it works!

File: config/initializers/warden.rb (hunk 1 above)
Comment (Kartones) [inline config/initializers/warden.rb]: Sadly there IS a `rack.session` (or N, only tried with one but might get N), is the warden proxy the one not present at testing :(

I tried playing with the `login_as` but it didn't created the proxy and also the "session" only lives for a single request so I can't properly check:
```
login with A
login with B
check A has no longer a session but B does
```

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5347/))

**Pull request merged**